src/plus/focus/enrichmentService.ts

```diff
@@ -209,22 +209,24 @@ export class EnrichmentService implements Disposable {
 	}
 }
 
-export function convertRemoteProviderToEnrichProvider(provider: RemoteProvider): EnrichedItemResponse['provider'] {
-	switch (provider.id) {
-		case 'azure-devops':
-			return 'azure';
-
-		case 'bitbucket':
-		case 'bitbucket-server':
-			return 'bitbucket';
+const supportedRemoteProvidersToEnrich: Record<string, EnrichedItemResponse['provider']> = {
+	'azure-devops': 'azure',
+	bitbucket: 'bitbucket',
+	'bitbucket-server': 'bitbucket',
+	github: 'github',
+	gitlab: 'gitlab',
+};
 
-		case 'github':
-			return 'github';
+export function convertRemoteProviderToEnrichProvider(provider: RemoteProvider): EnrichedItemResponse['provider'] {
+	return convertRemoteProviderIdToEnrichProvider(provider.id);
+}
```

Comment:
1. The `convertRemoteProviderToEnrichProvider` function was already existing, it encapsulated some logic, I have changed the implementation without changing the interface—a funcitn call. So I didn't need to change the existing code and won't need to change it in future if the implementation will be updated again.
2. The dictionary itself cannot replace the `convertRemoteProviderToEnrichProvider` function. It could replace `convertRemoteProviderIdToEnrichProvider`. But the function is more complex than just getting a value from the dictionary: it checks for existence and throws an error. With a constant I would need to repeat handling in all places where it's used.
3. The same new function `convertRemoteProviderIdToEnrichProvider` is used in a new place where I needed it and in the implementation of the old function `convertRemoteProviderToEnrichProvider` as a helper.